metadata_tools/common.py

```
@@ -0,0 +1,235 @@
+
+##########################################################################################
+# Argument parser
+##########################################################################################
+
+##########################################################################################
+# PathAction class
+##########################################################################################
+class PathAction(argparse.Action):
+    """Action method for path arguments.
+    """
+    def __call__(self, parser, namespace, values, option_string=None):
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__call__`

(ANN204)